inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/generic_eltwise_ref.cl

```diff
@@ -117,13 +117,8 @@ KERNEL(eltwise)(
     DO_ELTWISE;
 
 #if HAS_FUSED_OPS
-    #if ELTWISE_NO_PITCH_SAME_DIMS
-        FUSED_OPS_LINEAR;
-        OUTPUT_TYPE out = FUSED_OPS_RESULT_LINEAR;
-    #else
-        FUSED_OPS_TENSOR;
-        OUTPUT_TYPE out = FUSED_OPS_RESULT_TENSOR;
-    #endif
+    FUSED_OPS;
+    OUTPUT_TYPE out = FUSED_OPS_RESULT;
 #else
     #define out res
```

Comment:
Why do we use `#define` here? I think `OUTPUT_TYPE out = res;` will be more satisfy to the code style.

Reply:
This approach is claimed to be faster for some reason. I don't have a goal to refactor eltwise kernel within this PR